### PR TITLE
feat(StyleGuideProvider): context api

### PR DESCRIPTION
**Pull request by AlaaHamidah**
In order to centralise the tenant configuration, this PR is to remove the using of `process.env` (for tenants) and replace it with the value that comes from `srp-react`
please refer to https://trello.com/c/nAyAox9A for more information

this will combined later with @ZhenYongSeekAsia changes on `srp-react` [#1093](https://github.com/seekinternational/srp-react/pull/1093)

You can test this by changing the tenant in here 
image

### CODE REVIEW

**Comment by ZhenYongSeekAsia** [comment]
preview: [seek-asia-style-guide--dc03ee27d57f0ea346e4894b65b8db4f31af3eaf.surge.sh](seek-asia-style-guide--dc03ee27d57f0ea346e4894b65b8db4f31af3eaf.surge.sh)

File: docs/src/components/App/App.js [hunk 1]
```diff
@@ -27,27 +27,43 @@ const demoRoutes = demoSpecs.map(demoSpec => {
...
-      <Route path="/sandbox" render={() => null} />
-      <Route component={Footer} />
-    </Switch>
-  </StyleGuideProvider>
-);
+export default class App extends Component {
+  constructor() {
+    super();
+
+    this.changeTenant = this.changeTenant.bind(this);
```
Comment (cm-seekasia) [inline docs/src/components/App/App.js]: Can be removed this line

File: docs/src/components/App/Header/Header.js [hunk 2]
```diff
@@ -34,6 +41,8 @@ export default class Header extends Component {
   };
 
   render() {
+    const { changeTenant } = this.props;
+    console.log(changeTenant);
```
Comment (cm-seekasia) [inline docs/src/components/App/Header/Header.js]: plz remove

File: docs/src/components/App/Header/Header.js [hunk 3]
```diff
@@ -139,5 +239,6 @@ export default class Header extends Component {
 }
 
 Header.propTypes = {
-  tenant: PropTypes.string
+  tenant: PropTypes.string,
```
Comment (cm-seekasia) [inline docs/src/components/App/Header/Header.js]: no more?

**Comment by youwei91** [comment]
in the preview the header banner color is gone
Screen Shot 2019-03-19 at 12 45 26 PM

**Comment by ChanJit** [comment]
found that you do have an error on circle ci 
`Without `from` option PostCSS could generate wrong source map and will not find Browserslist config. Set it to CSS file path or to `undefined` to prevent this warning.
Too long with no output (exceeded 10m0s)`

**Comment by ZhenYongSeekAsia** [comment]
@youwei91 yes, because i thinking to reduce the effort for refactor

**Comment by youwei91** [comment]
I try to locally install this PR into https://github.com/seekinternational/srp-react/pull/1093, and seem like the button still gone off, is that expected or I miss out something?
Screen Shot 2019-03-19 at 5 04 13 PM

**Comment by ZhenYongSeekAsia** [comment]
@youwei91 you using which branch? you need to add the country prop in StyleGuideProvider in srp-react.

**Comment by youwei91** [comment]
> @youwei91 you using which branch? you need to add the country prop in StyleGuideProvider in srp-react.
I tried is master & your [PR branch](https://github.com/seekinternational/srp-react/pull/1093). This PR is to match to your [PR](https://github.com/seekinternational/srp-react/pull/1093) to overcome the button color issue right? correct me if I'm wrong

**Pull request merged**